conf/logs.json

```diff
@@ -1236,6 +1243,31 @@
       "json_path": "logEvents[*].extractedFields"
     }
   },
+  "cloudwatch:rds_aurora": {
+    "schema": {
+      "timestamp": "integer",
+      "serverhost": "string",
+      "username": "string",
+      "host": "string",
+      "connectionid": "integer",
+      "queryid": "integer",
+      "operation": "string",
+      "database": "string",
+      "object": "string",
+      "retcode": "integer"
+    },
+    "parser": "csv",
+    "configuration": {
+      "envelope_keys": {
+        "logGroup": "string",
+        "logStream": "string",
+        "messageType": "string",
+        "owner": "string",
+        "subscriptionFilters": []
+      },
+      "json_path": "logEvents[*].message"
```

Comment:
Does `logEvents` need to be specified in the `envelope_keys` or anywhere else in the schema config?

Reply:
No - good catch though!